packages/main/src/ComboBoxGroupItem.ts

```diff
@@ -34,13 +30,15 @@ class ComboBoxGroupItem extends UI5Element implements IComboBoxItem {
 		/**
 		 * Indicates whether the item is focssed
 		 * @protected
+		 * @type {boolean}
 		 */
 		@property({ type: Boolean })
 		focused!: boolean;
 
 		/**
 		 * Used to avoid tag name checks
 		 * @protected
+		 * @type {boolean}
```

Comment:
I see. So, the same change should be done for the other getters, as this is the only one enhanced in that way.

Reply:
All currently documented getters in the scope of this PR are now with @type and @default. This will be reflected in the other PRs as well. I haven't documented @private getters or ones without documentation so far.